integration/container/run_linux_test.go

```diff
@@ -401,3 +401,62 @@ func TestSeccomp(t *testing.T) {
 		}
 	}
 }
+
+func TestCgroupRW(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
+
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	type testCase struct {
+		ops              []func(*container.TestContainerConfig)
+		expectedErrMsg   string
+		expectedExitCode int
+	}
+	testCases := []testCase{
+		{
+			ops: nil,
+			// no err msg, so, success
```

Comment:
Looks like this one needs `expectedExitCode: 1` since its using the default case which should be non-writable.